languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -26,10 +26,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!ENTITY common_english_nouns_feminine "University|Society|League|Corporation|Association|Gallery|Navy|Area|Cup|Foundation|Conference|Tool|Union|Agency|Company|Force|Licence|License|Station">
```

Comment:
You have removed "haïr", but some dictionaries say it is "h aspiré". https://fr.wiktionary.org/wiki/ha%C3%AFr

Reply:
Definitely, e.g. to differentiate "j'ai" from ["je hais"](https://leconjugueur.lefigaro.fr/conjugaison/verbe/ha%EFr.html)